src/runtime_src/core/tools/common/Table2D.cpp

```diff
@@ -22,93 +11,120 @@
 #include <algorithm>
 #include <boost/format.hpp>
 
-Table2D::Table2D(const std::vector<HeaderData>& headers)
+Table2D::Table2D(const std::vector<HeaderData>& headers) : m_inter_entry_padding(2)
 {
-    for (auto& header : headers)
-        addHeader(header);
+  for (auto& header : headers)
+      addHeader(header);
 }
 
 void
 Table2D::addEntry(const std::vector<std::string>& entry)
 {
-    if (entry.size() < m_table.size())
-        throw std::runtime_error("Table2D - Entry data is smaller than table.\n");
-    else if (entry.size() > m_table.size())
-        throw std::runtime_error("Table2D - Entry data is larger than table.\n");
+  if (entry.size() < m_table.size())
+    throw std::runtime_error("Table2D - Entry data is smaller than table.\n");
```

Comment:
It is great that we have this exception.  Add some more information to help understand the underlying issue.  In this case, also report the entry and m_table sizes.